plugins/meta/flannel/flannel.go

```diff
@@ -57,19 +57,27 @@ type NetConf struct {
 type subnetEnv struct {
 	nw     *net.IPNet
 	sn     *net.IPNet
+	ip6Nw  *net.IPNet
+	ip6Sn  *net.IPNet
 	mtu    *uint
 	ipmasq *bool
 }
 
 func (se *subnetEnv) missing() string {
 	m := []string{}
 
-	if se.nw == nil {
+	if se.nw == nil && se.ip6Nw == nil {
 		m = append(m, "FLANNEL_NETWORK")
 	}
-	if se.sn == nil {
+	if se.sn == nil && se.ip6Sn == nil {
 		m = append(m, "FLANNEL_SUBNET")
 	}
+	if se.ip6Nw == nil && se.nw == nil {
```

Comment:
Why it's a separate if? Conditions are the same as in line 69, just in reverse order, right?

Reply:
It's my mistake, This if conditions have repeated. I will updated it.